app/js/filterFunctions.js

```diff
@@ -35,6 +35,7 @@ document.getElementById('startDate').addEventListener("change", ()=>{
     let dates = setDate()
     if (dates[0] <= dates[1]) {
         updateScoreTable()
+        updateChart()
```

Comment:
Does this not now need to be given users?

Reply:
Actually, it no longer needs to be called outside updateScoreTable at all. Thanks for the spot!